repository.yaml

```diff
@@ -1,5 +1,4 @@
 repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
 test-repositories:
   - contact-hmrcdeskpro-performance-tests
-  - platform-performance-tests
```

Comment:
PR Bot -> uk.gov.hmrc.sbt-auto-build can be updated to 3.21.0

Reply:
going to be naughty and skip this because I missed it yesterday and haven't got time to do it now :cat-typing: